public/js/products.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = {
+  'mozilla-concrete-brick': require('json!mozilla-concrete-brick'),
+  'mozilla-concrete-mortar': require('json!mozilla-concrete-mortar'),
```

Comment:
is there a better way to link products? As is, we'd have to edit this dictionary each time we add/remove products. Ideally we'd want to just update the payments-config module and be done.

Would something similar to this work?

    module.exports = require('json!products-config')